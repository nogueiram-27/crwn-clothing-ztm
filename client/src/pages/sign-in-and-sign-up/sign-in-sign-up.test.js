import React from 'react'
import { shallow } from 'enzyme'

import SignInAndSignUpPage from './sign-in-and-sign-up.component'; 

describe('SignInAndSignUpPage', () => {

    it('should render SignInAndSignUpPage component', () => {
        expect(shallow(<SignInAndSignUpPage />)).toMatchSnapshot();
    })
})