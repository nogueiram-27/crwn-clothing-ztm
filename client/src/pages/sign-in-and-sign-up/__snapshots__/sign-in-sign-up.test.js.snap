// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInAndSignUpPage should render SignInAndSignUpPage component 1`] = `ShallowWrapper {}`;
